xml/hwbp_registers.xml

```diff
@@ -378,6 +378,9 @@
             though the load will not update its destination register. Debuggers
             should consider this when setting such breakpoints on, for example,
             memory-mapped I/O addresses.
+
+            It is undefined what should happen when the instruction in question
```

Comment:
Saying that the behavior is "undefined" means that the possibilities are architecturally unbounded.  Any behavior would be allowed - which is both undesirable and useless to some debugging code with such instructions.

Conversely, I would expect people that care about the Zc* instructions would (rightly) object to freeze (let alone ratification) of Debug 1.0 since it would be unusuable on embedded software that makes use of these instructions.

Reply:
What is the right term here? Unspecified?

`mcontrol` triggers are already deprecated, but debuggers should know about them so they're in the spec. New hardware implementations should implement mcontrol6 instead, where the behavior is specified. Is that OK, or should I invent something for people insisting on mcontrol triggers with Zc*?